packages/oid4vci-demo-frontend/src/pages/SSIIdentityVerificationPage/index.tsx

```diff
@@ -0,0 +1,102 @@
+import React, {ReactElement} from 'react';
+import {useMediaQuery} from 'react-responsive';
+import {useTranslation} from 'react-i18next';
+import {useFlowRouter} from '../../router/flow-router';
+import {SSIIdentityVerificationPageConfig} from '../../ecosystem/ecosystem-config';
+import {NonMobile} from '../../index';
+import style from './index.module.css'
+import InputField from "../../components/InputField";
+import SSIPrimaryButton from "../../components/SSIPrimaryButton";
+import {useLocation} from "react-router-dom";
+
+const SSIIdentityVerificationPage: React.FC = (): ReactElement => {
+    const {t} = useTranslation()
+    const flowRouter = useFlowRouter<SSIIdentityVerificationPageConfig>()
+    const pageConfig = flowRouter.getPageConfig();
+    const isTabletOrMobile = useMediaQuery({query: '(max-width: 767px)'})
+    const location = useLocation();
+    const state = location.state
+
+    const onContinue = async (): Promise<void> => {
+        await flowRouter.nextStep({payload: state})
```

Comment:
No longer required, you can also maybe remove lines 17 & 18 if not used anywhere else

Reply:
fixed